app/presenters/speedy_af/proxy/media_object.rb

```diff
@@ -29,6 +29,10 @@ def to_param
     id
   end
 
+  def published?
+    !avalon_publisher.blank?
```

Comment:
Would it be better to use `avalon_publisher.present?` instead of negating a `blank?` check?